server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -1606,13 +1606,14 @@ private void initStorageEngineAllocate(Properties properties) {
 
     String allocationRatioForWrite = properties.getProperty("write_memory_proportion", "19:1");
     proportions = allocationRatioForWrite.split(":");
-    int proportionForMemTable = Integer.parseInt(proportions[0].trim());
+    int proportionForMemTable = Integer.parseInt(proportions[0].replace("\\", "").trim());
```

Comment:
replace("\\", "") is for what?

Reply:
<img width="835" alt="image" src="https://user-images.githubusercontent.com/25913899/223011112-831f0e81-edd0-4825-85b6-c01a92135129.png">

The serialize method of property will add `\` for `:`.